packages/icons-angular/package.json

```diff
@@ -4,41 +4,48 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "lint": "ng lint",
+    "lint": "ng lint --project ant-design-icons-angular",
     "test": "ng test --no-watch --no-progress --browsers=ChromeHeadlessCI --code-coverage",
-    "build": "ng build --prod",
-    "build:lib": "npm run generate && ng build ant-design-icons-angular --configuration production && npm run copy:assets",
+    "build": "ng build --project ant-design-icons-angular --configuration production",
```

Comment:
This script is for building the demo webpage. Please do not change this.

Reply:
ng build --prod is deprecated.

Reply:
As is default project. It has to be named.

Reply:
If I chose the wrong project, I'll fix it

Reply:
Looks good!